libass/ass_render.c

```diff
@@ -1055,8 +1055,8 @@ get_outline_glyph(ASS_Renderer *priv, GlyphInfo *info)
             ass_font_set_transform(info->font, info->scale_x, info->scale_y);
             FT_Glyph glyph =
                 ass_font_get_glyph(info->font,
-                        info->symbol, info->face_index, info->glyph_index,
-                        priv->settings.hinting, info->flags);
+                        info->face_index, info->glyph_index,
+                        priv->settings.hinting, key.u.glyph.flags);
```

Comment:
Why is this still changed from `info->flags` to `key.u.glyph.flags`? A previous version of this commit modified the hash key flags, but this version no longer does so, so this seems unncessary.

Reply:
On that note, was that change made in response to some of my comments that I’m now forgetting, and does it have anything to do with the rendering of line breaks (which are now mentioned in the commit message), or did you just realize there was no need to do it in the cache key?

Reply:
Yes, that looks like artifact from commit editing. I'll revert back that line.